psydac/core/bsplines.py

```diff
@@ -78,14 +83,46 @@ def find_span( knots, degree, x ):
     return span
 
 #==============================================================================
-def basis_funs( knots, degree, x, span ):
+
```

Comment:
Please remove this empty line before the function definition

Reply:
Fixed in https://github.com/pyccel/psydac/commit/c9eac22256b58bb6b2138192ad3cc5766ae2d143